lib/python/treadmill/cli/admin/install/openldap.py

```diff
@@ -51,7 +52,7 @@ def openldap(ctx, gssapi, rootpw, owner, suffix, uri, masters,
             run_script = os.path.join(dst_dir, 'bin', 'run.sh')
 
         ctx.obj['PARAMS']['env'] = env
-        ctx.obj['PARAMS']['treadmillid'] = owner
+        ctx.obj['PARAMS']['treadmillid'] = ','.join(owner)
```

Comment:
i rather not overload treadmillid with the list. it has meaning, and we are talking about additional ldap owners.

to be safe, make it explicit --ldap-admins, make a special param in the context, and modify install templates to concatenate both.